src/GLideNUI/Settings.cpp

```diff
@@ -411,6 +413,7 @@ void saveCustomRomSettings(const QString & _strIniFolder, const char * _strRomNa
 	WriteCustomSetting(video, windowedHeight);
 	WriteCustomSetting(video, fullscreenRefresh);
 	WriteCustomSetting(video, multisampling);
+	WriteCustomSetting(video, maxMultiSampling);
```

Comment:
I don't think that this is necessary. maxMultiSampling can't be per-game setting.